docs/resources/item.md

```diff
@@ -66,7 +66,7 @@ resource "onepassword_item" "demo_db" {
 
 ### Optional
 
-- `category` (String) The category of the item. One of ["login" "password" "database" "secure_note"]
+- `category` (String) The category of the item. One of ["login" "password" "database" "secure_note" "document"]
```

Comment:
This documentation is inaccurate and can lead to confusion for customers. 😕

Reply:
fixed